services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from typing import List
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+    return movie_session
+
+
+def get_movies_sessions(
+        session_date: datetime = None
+) -> List[MovieSession]:

```

Comment:
Correct annotation

QuerySet[MovieSession]

Reply:
The comment is not resolved